outport/process/alteredaccounts/shared/shared.go

```diff
@@ -14,9 +14,10 @@ var errInvalidAlteredAccountsOptions = errors.New("invalid altered accounts opti
 
 // AlteredAccountsOptions holds some configurable parameters to be used for extracting the altered accounts
 type AlteredAccountsOptions struct {
-	WithCustomAccountsRepository bool
-	AccountsRepository           state.AccountsRepository
-	AccountQueryOptions          api.AccountQueryOptions
+	WithCustomAccountsRepository  bool
+	AdditionalAlteredAccountsData bool
```

Comment:
might rename to `WithAdditionalOutportData`

Reply:
done